src/jmh/java/hu/akarnokd/comparison/scrabble/ShakespearePlaysScrabbleWithCyclopsReactOpt.java

```diff
@@ -49,145 +66,99 @@
     )
     @Fork(1)
     public List<Entry<Integer, List<String>>> measureThroughput() throws InterruptedException {
-
-        //  to compute the score of a given word
-        Function<Integer, Integer> scoreOfALetter = letter -> letterScores[letter - 'a'];
-
-        // score of the same letters in a word
-        Function<Entry<Integer, MutableLong>, Integer> letterScore =
-                entry ->
-                        letterScores[entry.getKey() - 'a'] *
-                        Integer.min(
-                                (int)entry.getValue().get(),
-                                scrabbleAvailableLetters[entry.getKey() - 'a']
-                            )
-                    ;
-
-
-        Function<String, ReactiveSeq<Integer>> toIntegerIx =
-                string -> chars(string);
-
-        // Histogram of the letters in a given word
-        Function<String, ReactiveSeq<HashMap<Integer, MutableLong>>> histoOfLetters =
-                word ->  toIntegerIx.apply(word)
-                            .scanLeft(new HashMap<Integer, MutableLong>(), (map, value) -> {
-                                MutableLong newValue = map.get(value) ;
-                                if (newValue == null) {
-                                    newValue = new MutableLong();
-                                    map.put(value, newValue);
-                                }
-                                newValue.incAndSet();
-                                return map;
-                            })
-                        .takeRight(1);
-                 ;
-
-        // number of blanks for a given letter
-        Function<Entry<Integer, MutableLong>, Long> blank =
-                entry ->
-                        Long.max(
-                            0L,
-                            entry.getValue().get() -
-                            scrabbleAvailableLetters[entry.getKey() - 'a']
-                        )
-                    ;
-
+                                               
+         // Histogram of the letters in a given word
+        Function<String, ReactiveSeq<Map.Entry<Integer, Long>>> histoOfLetters =
```

Comment:
Hey David,

Thanks for the feedback. I am going to state my position directly below for clarity (trust me I mean no offence - I hold you in very high esteem, but I need to be clear that these benchmarks are not consistent and are thus biased).

* I'll make the changes you have requested when we get equivalent changes to the Streams benchmark - otherwise I am accepting and endorsing a benchmark that is heavily biased in favour of Streams and against cyclops-react.  I hope you appreciate that I can not do this, the bar can-not be higher for one than the other.

* I appreciate that these benchmark mean something specific and useful to you - but they are also being published to the public at large, that means they need to be consistent and fair. The benchmark you have published shows a 5X (or more) performance penalty for cyclops-react versus JDK 8 Streams, I have demonstrated clearly at this point that no such performance penalty exists.

I hope we can resolve this one way or the other. Either we make the benchmark fair, or we remove cyclops-react from it.

Thanks,
John